### PR TITLE
[HOTFIX] Changes group metadata uris from paths to member names

### PR DESCRIPTION
**Pull request by ktsitsi**

### CODE REVIEW

File: tiledb/bioimg/converters/base.py [hunk 1]
```diff
@@ -227,7 +227,7 @@ def to_tiledb(
 
                 # Store layer mapping with shape value
                 meta_kvstore = {
-                    "uri": uri,
+                    "uri": os.path.basename(uri),
```
Comment (gsakkis) [inline tiledb/bioimg/converters/base.py:230]: - Key should be renamed to to "name"
- `os.path.basename(uri)` is also referenced later in `group.add()`, better assign it to a variable.

**Review by gsakkis — COMMENTED** [review]
LGTM but should be postponed because https://github.com/TileDB-Inc/TileDB-BioImaging/pull/61/ has duplicated this logic. So
1. Merge https://github.com/TileDB-Inc/TileDB-BioImaging/pull/61/
2. Refactor base.py to DRY-fy it (I'm working on it atm)
3. Rebase/update this PR

File: tiledb/bioimg/converters/base.py (hunk 1 above)
Comment (ktsitsi) [reply tiledb/bioimg/converters/base.py]: For the second point `group.add()` is in an external `for` loop so there are no related. Reverted the change cause the tests were failing.

**Comment by gsakkis** [comment]
Subsumed by https://github.com/TileDB-Inc/TileDB-BioImaging/pull/68

**Pull request closed without merging**